docs/IgorFromUA/login.html

```diff
@@ -0,0 +1,98 @@
+<!doctype html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport"
+            content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+        <meta http-equiv="X-UA-Compatible" content="ie=edge">
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+        <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+        <link rel="stylesheet" type="text/css" href="styles/styles.css">
+        <link rel="stylesheet" type="text/css" href="styles/header.css">
+        <link rel="stylesheet" type="text/css" href="styles/login.css">
+        <link rel="stylesheet" type="text/css" href="styles/custom-checkbox.css">
+        <link rel="stylesheet" type="text/css" href="styles/footer.css">
+        <title>GamesProject</title>
+    </head>
+    <body class="wrapper-all">
+        <div class="wrapper">
+            <header class="header container">
+                <img class="header__logo" src="./src/logo.png" width="55" height="58" alt="logo masters academy">
+                <nav class="navigation">
+                    <div class="navigation__menu-icon">
+                        <svg width="34" height="19" viewBox="0 0 34 19">
+                            <rect width="34" height="3.5" rx="1" fill="#FFF"></rect>
+                            <rect y="8" width="34" height="3.5" rx="1" fill="#FFF"></rect>
+                            <rect y="16" width="34" height="3.5" rx="1" fill="#FFF"></rect>
+                        </svg>
+                    </div>
+                    <ul class="navigation__list">
+                        <li class="navigation__item">
+                            <a class="navigation__link" href="./calculator.html">Calculator</a>
+                        </li>
+                        <li class="navigation__item">
+                            <a class="navigation__link" href="./games.html">Games</a>
+                        </li>
+                        <li class="navigation__item">
+                            <a class="navigation__link" href="./index.html">About</a>
+                        </li>
+                        <li class="navigation__item">
+                            <a class="navigation__link navigation__link_active" href="./login.html">Login</a>
+                        </li>
+                    </ul>
+                </nav>
+            </header>
+            <main class="container">
+                <section class="login">
+                    <h1 class="login__title">Login page</h1>
+                    <form action="./successful-login.html">
+                        <label class="label-input">Email
+                            <input id="email-input" type="text" class="login__input" placeholder="Enter your email address">
```

Comment:
It could enhance UI
```suggestion
                            <input id="email-input" type="email" class="login__input" placeholder="Enter your email address">
```